jms/src/test/scala/akka/stream/alpakka/jms/JmsSpec.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.jms
+
+import java.net.ServerSocket
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import org.apache.activemq.broker.BrokerService
+import org.scalatest._
+import org.scalatest.concurrent.ScalaFutures
+
+abstract class JmsSpec
+    extends WordSpec
+    with Matchers
+    with BeforeAndAfterAll
+    with BeforeAndAfterEach
+    with ScalaFutures {
+
+  implicit val system = ActorSystem(this.getClass.getSimpleName)
+  implicit val materializer = ActorMaterializer()
+
+  override protected def afterAll(): Unit =
+    system.terminate()
```

Comment:
`TestKit.shutdown(system)` will terminate and block with a timeout, use that instead